examples/resolved_cfd-dem/sphere_carreau_with_sharp_interface/sphere_carreau_with_sharp_interface.prm

```diff
@@ -0,0 +1,204 @@
+	    end
+    end
+  end
+end
+
+#---------------------------------------------------
+# Mesh
+#---------------------------------------------------
+subsection mesh
+    set type                 = dealii
+    set grid type            = subdivided_hyper_rectangle
+    set grid arguments       = 2,1,1 : -18,-15,-15 : 42,15,15 : true
+    set initial refinement   = 4
+end
+
+subsection  box refinement
+    set initial refinement   = 3
+    subsection mesh
+    set type                 = dealii
+    set grid type            = subdivided_hyper_rectangle
+    set grid arguments       = 1,1,1: -2,-2,-2 : 6,2,2 : true
+    set initial refinement   = 0
+    end
+end
+
+
+
+# --------------------------------------------------
+# Boundary Conditions
+#---------------------------------------------------
+subsection boundary conditions
+set number                  = 5
+    subsection bc 0
+	set id = 0
+        set type              = function
+        subsection u
+            set Function expression = 1
+        end
+        subsection v
+            set Function expression = 0
+        end
+        subsection w
+            set Function expression = 0
+        end
+    end
+    subsection bc 1
+	set id = 2
+        set type              = slip
+    end    
+    subsection bc 2
+	set id = 3
+        set type              = slip
+    end
+    subsection bc 3
+	set id = 4
+        set type              = slip
+    end
+    subsection bc 4
+	set id = 5
+        set type              = slip
+    end
+end
+
+# --------------------------------------------------
+# IB particules
+#---------------------------------------------------
+subsection particles
+	set number of particles = 1
+	set stencil order = 2
+	set length ratio  = 1
+	set refine mesh inside radius factor = 0.85
+	set refine mesh outside radius factor = 1.3
+	set initial refinement = 2
+	set integrate motion = false
+	set assemble Navier-Stokes inside particles = false    
+	subsection particle info 0
+	subsection position
+	    set Function expression =0;0;0
+	end
+	    set pressure x =0.00001
+	    set pressure y =0.00001
+	    set pressure z =0.00001
+	    set radius = 0.5
+    end
+end
+# --------------------------------------------------
+# Mesh Adaptation Control
+#---------------------------------------------------
+subsection mesh adaptation
+  set type                    = kelly
+  set fraction coarsening     = 0.1
+  set fraction refinement     = 0.4
+  set fraction type	      = number
+  set frequency            = 1
+  set max number elements     = 8000000
+  set min refinement level    = 0
+  set max refinement level    = 11
+  
+  set variable		      = velocity
+end
+# --------------------------------------------------
+# Non-Linear Solver Control
+#---------------------------------------------------
+subsection non-linear solver
+  set verbosity               = verbose
+  set tolerance               = 1e-5
+  set max iterations          = 20
+  set residual precision      = 5
```

Comment:
why the residual precision?

Reply:
It could be smaller, I could change it, though it wouldn't change the forces/torque output much

Reply:
No worries, I was just wondering why you were specifying a specific value here of 5, generally we leave the default value, but if you put it, it's allright. Just explain why in the example. This way people understand why you are doing what you are doing.